src/otx/algorithms/common/adapters/mmcv/configurer.py

```diff
@@ -416,13 +416,15 @@ def configure_hooks(
         # Update adaptive repeat
         if not self.training:
             remove_from_configs_by_type(cfg.custom_hooks, "AdaptiveRepeatDataHook")
+            return
         for custom_hook in cfg.custom_hooks:
             if custom_hook["type"] == "AdaptiveRepeatDataHook":
                 data_cfg = cfg.get("data", {})
                 bs = data_cfg.get("train_dataloader", {}).get("samples_per_gpu", None)
                 bs = bs if bs is not None else data_cfg.get("samples_per_gpu", 0)
                 custom_hook["train_batch_size"] = bs
-                custom_hook["n_train_data"] = len(data_cfg.get("train", {}).get("otx_dataset", []))
+                custom_hook["train_data_size"] = len(self.get_data_cfg(cfg, "train"))
```

Comment:
len(train_data_cfg) might not what you want.

Reply:
Thanks :) I edited.

Reply:
For some reason, I decided to avoid using `get_data_cfg`. When `cfg` doesn't include the `data` field, `get_data_cfg` raises an error. However, there are some unit tests regarding the `custom_hook` that don't consider the `cfg.data`. So, it raises an error. To avoid it, I think using the data_cfg.get("train", {}).~~ is the better choice.